src/components/panel/panel.spec.js

```diff
@@ -868,6 +895,76 @@ describe('$mdPanel', function() {
       });
     });
 
+    describe('CSS class logic:', function() {
+      it('should add a class to the container/wrapper when toElement=false',
+        function() {
+        openPanel(DEFAULT_CONFIG);
```

Comment:
nit: indentation here is off. openPanel and the rest of the code still needs to be indented under function, like this:
```
it('should add a class to the container/wrapper when toElement=false',
  function() {
    openPanel(DEFAULT_CONFIG);
    ...
    expect(PANEL_WRAPPER_CLASS).toHaveCLASS('my-class');
  });